### ISSUE TITLE
Add new WCAG 2.2 Glossary terms, definitions, and interpretation guidance

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag2ict#194]
Add the [new terms in WCAG 2.2](https://github.com/w3c/wcag/tree/main/guidelines/terms/22). Terms will be added as the SCs that use them are added.

- [x] [cognitive function test](https://www.w3.org/TR/WCAG22/#dfn-cognitive-function-test) (Uses "websites") **3.3.8 Accessible Authentication (level AA)** and **3.3.9 Accessible Authentication (Enhanced) (Level AAA)**
- [x] [dragging movement](https://www.w3.org/TR/WCAG22/#dfn-dragging-movements) (no WCAG2ICT interpretation needed) **2.5.7 Dragging Movements (Level AA)**
- [x] [encloses](https://www.w3.org/TR/WCAG22/#dfn-enclose) (no WCAG2ICT interpretation needed). This is not referenced by any WCAG SC, so should say that in WCAG2ICT.
- [x] [focus indicator](https://www.w3.org/TR/WCAG22/#dfn-focus-indicator) (no WCAG2ICT interpretation needed) **2.4.11 Focus Appearance (Level AAA)** and **2.4.7 Focus Visible (Level AA)**
- [x] [minimum bounding box](https://www.w3.org/TR/WCAG22/#dfn-minimum-bounding-box) Linked to by **2.5.8 Target Size (Level AA)** and by the term "perimeter" used in **2.4.11 Focus Appearance (Level AAA)**. Also need to remove "bounding box" from the terms that apply to all SC, as this term was replaced by "minimum bounding box".
- [x] [perimeter](https://www.w3.org/TR/WCAG22/#dfn-perimeter) (no WCAG2ICT interpretation needed) **2.4.11 Focus Appearance (Level AAA)**
- [x] [pointer input](https://www.w3.org/TR/WCAG22/#dfn-pointer-inputs) (no WCAG2ICT interpretation needed) **2.5.8 Target Size (Minimum) (Level AA)** and **2.5.5 Target Size (Enhanced) (Level AAA)**
- [x] [target](https://www.w3.org/TR/WCAG22/#dfn-targets) (Definition uses "page") 2.5.5 Target Size (Level AAA) added in WCAG 2.1, **2.5.8 Target Size (Minimum) (Level AA)** (added in WCAG 2.2), **2.5.5 Target Size (Enhanced) (Level AAA)** changed from WCAG 2.1.

**Comment by maryjom** [comment]
Added encloses, focus indicator, minimum bounding box, and perimeter in PR 282.

**Comment by maryjom** [comment]
Started [survey](https://www.w3.org/2002/09/wbs/55145/WCAG2ICT-term-definitions/results) on "encloses", "focus indicator", "minimum bounding box" and "perimeter" - definitions were approved by the TF. Accessible Authentication is still in progress.

**Comment by maryjom** [comment]
This work is all done. The AG WG has reviewed the definitions as we have worked through them. The AG WG review in December 2023 covered "encloses", "focus indicator", "minimum bounding box" and "perimeter", all of which were approved. The last new term "cognitive function test" is under review as part of Issue #368 which concludes on 28 May. Closing this issue as this was for the development of the content. Any further definitions updates will be tracked either in 368 or in new issues.
